### ISSUE TITLE
it is locon? not lora?

### ISSUE THREAD

**Issue by henbucuoshanghai** [kohya-ss/sd-scripts#785]
self.lora_down = torch.nn.Conv2d(in_dim, self.lora_dim, kernel_size, stride, padding, bias=False)
